playbooks/openshift-glusterfs/upgrade.yml

```diff
@@ -1,4 +1,15 @@
 ---
+- name: Upgrade is in tech preview
+  hosts: all
+  gather_facts: False
+  become: False
+  tasks:
+    - name: Check if user acknowledged that upgrade playbook is in tech preview
+      fail:
+        msg: "Upgrade playbook is in tech preview, please define var openshift_storage_glusterfs_upgrade_in_tech_preview=true to proceed"
```

Comment:
`openshift_storage_gluster_update_techpreview`, similar to `openshift_logging_es5_techpreview` we had in 3.10?